From cccd4a1153f09b8db6ef50515ca49b1d58d9604c Mon Sep 17 00:00:00 2001
From: Tony Fischetti <tony.fischetti@gmail.com>
Date: Mon, 6 Jul 2020 17:44:53 -0400
Subject: [PATCH] temporary fix for regression (auto instatiation of eeproms)

---
 drivers/i2c/busses/i2c-i801.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/i2c/busses/i2c-i801.c b/drivers/i2c/busses/i2c-i801.c
index fea644921a76..4f333889489c 100644
--- a/drivers/i2c/busses/i2c-i801.c
+++ b/drivers/i2c/busses/i2c-i801.c
@@ -1318,12 +1318,6 @@ static void i801_probe_optional_slaves(struct i801_priv *priv)
 
 	if (is_dell_system_with_lis3lv02d())
 		register_dell_lis3lv02d_i2c_device(priv);
-
-	/* Instantiate SPD EEPROMs unless the SMBus is multiplexed */
-#if IS_ENABLED(CONFIG_I2C_MUX_GPIO)
-	if (!priv->mux_drvdata)
-#endif
-		i2c_register_spd(&priv->adapter);
 }
 #else
 static void __init input_apanel_init(void) {}
-- 
2.20.1

