From 862b3bc4c6b6413bb46b60610f494deffad4cdf3 Mon Sep 17 00:00:00 2001
From: Tony Fischetti <tony.fischetti@gmail.com>
Date: Mon, 15 Jun 2020 07:58:36 -0400
Subject: [PATCH] change swappiness default from 60 to 13

---
 mm/vmscan.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/vmscan.c b/mm/vmscan.c
index b6d84326bdf2..bc16539360c2 100644
--- a/mm/vmscan.c
+++ b/mm/vmscan.c
@@ -169,7 +169,7 @@ struct scan_control {
 /*
  * From 0 .. 200.  Higher means more swappy.
  */
-int vm_swappiness = 60;
+int vm_swappiness = 13;
 /*
  * The total number of pages which are beyond the high watermark within all
  * zones.
-- 
2.20.1

