From 3d1f8e89d4d2bf6db04d74c4ab6ee21b3e0c869b Mon Sep 17 00:00:00 2001
From: Tony Fischetti <tony.fischetti@gmail.com>
Date: Thu, 12 Mar 2020 15:39:35 -0400
Subject: [PATCH 6/7] change swappiness default from 60 to 15

---
 mm/vmscan.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/vmscan.c b/mm/vmscan.c
index af4b2b3d4e0d..576d22956be5 100644
--- a/mm/vmscan.c
+++ b/mm/vmscan.c
@@ -177,7 +177,7 @@ struct scan_control {
 /*
  * From 0 .. 100.  Higher means more swappy.
  */
-int vm_swappiness = 60;
+int vm_swappiness = 15;
 /*
  * The total number of pages which are beyond the high watermark within all
  * zones.
-- 
2.20.1

